### ISSUE TITLE
Photo metadata

### ISSUE THREAD

**Issue by Gregory108** [umonkey/treemap#107]
I'v downloaded several photos (from "full" view), but none of them contained metadata about photo GPS, lens properties, photo settings, etc., which are desirable.

It is at all possible?

**Comment by umonkey** [comment]
Yes this is possible.  When resizing images, we can add metadata from the database.
